content/en/docs/setup/install/multicluster/simplified/index.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Simplified Multicluster Install [Experimental]
+description: Configure an Istio mesh spanning multiple Kubernetes clusters.
+weight: 1
```

Comment:
Do we want the `Experimental` mulitcluster item listed as the top item of the 4?